### DISCUSSION TITLE
0.5.8

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1423] [Announcements]
- Smoother navigation animation
- Option to hide explicit content
- Add squiggly slider
- Fix album artist error and add refetch album button
- Fix color mismatch
- Other fixes and enhancements

<hr />

If you encounter bugs, it's better to create an issue than reporting it here.
If you want some features, please create an issue.

**Comment by zenzer0s** [comment]
What's the difference between foss and full?

**Reply by z-huang** [reply]
https://github.com/z-huang/InnerTune/wiki/App-Versions

**Comment by postcanonical** [comment]
long time no see:) glad it happened! thank you!

**Comment by paula879** [comment]
Great update!. Just having an issue with the equalizer that it stops working after each song and i have to open the app again.

**Reply by z-huang** [reply]
Please make an issue!

**Comment by radish121** [comment]
is it not possible to have a normal slider :(

**Reply by Anonuser-code** [reply]
There are only 2 options in the settings. Unfortunately that's all.

**Reply by radish121** [reply]
hmmmm thats a bummer then, i miss the plain slider :( how can it have a squiggly slider but not a plain one 😔

**Reply by Anonuser-code** [reply]
> hmmmm thats a bummer then, i miss the plain slider :( how can it have a squiggly slider but not a plain one 😔

![IMG_20240822_193136.jpg](https://github.com/user-attachments/assets/724360d3-eaea-4e01-8345-94784cc2949c)

Have one

**Reply by radish121** [reply]
i actually meant the slider before the materal 3 update tho but hey its ok i guess, i just think the slider after the update kinda look bad

**Reply by z-huang** [reply]
The old slider is in the old material 3 library. Now the library has been updated, so the old slider is gone. Instead, I have to write a slider myself if many people really miss it. Though, I think people will get used to the new slider.

**Reply by radish121** [reply]
yeah i totally understand that

**Comment by Khazbs** [comment]
Squiggly slider?! Yes, please!! Hyped up and waiting for 0.5.8 to reach F-Droid!

**Reply by z-huang** [reply]
It's on IzzyOnDroid already.

**Reply by Khazbs** [reply]
It's in the official repo now!

**Comment by itszo-om** [comment]
Hey can you make for desktop version

**Reply by mamiyaso** [reply]
see [#962](https://github.com/z-huang/InnerTune/discussions/962) in discussions

**Reply by z-huang** [reply]
Yes. I've already put so much effort in Android version, and I don't have extra time to make a desktop version.

**Comment by itszo-om** [comment]
Is there such an another  app for desktop?

**Reply by ruskcoder** [reply]
Use SpMp https://github.com/toasterofbread/spmp/releases/tag/v0.4.0

**Reply by itszo-om** [reply]
It's safe ?

**Reply by ruskcoder** [reply]
Yes! Open source as well!

**Reply by itszo-om** [reply]
Doesn't work in Windows 7?

**Comment by hantu152** [comment]


**Reply by ruskcoder** [reply]
Create an issue for it. 
also song name...

**Comment by Smgeuk** [comment]
Hay un error molesto a la hora de mover canciones dentro dela lista de reproducción, ya que las canciones se inician a desorganizar.

Aqui adjunto se ve el problema :(

https://github.com/user-attachments/assets/21972e26-2c9c-481a-ae95-ea1cb0a7c7a5

**Reply by z-huang** [reply]
English please.

**Reply by Khazbs** [reply]
Crystal Castles 🤩

**Reply by Smgeuk** [reply]
> English please.

There is an annoying bug when moving songs within the playlist, as the songs start to get disorganized.

**Reply by z-huang** [reply]
Maybe it's due to reordering too frequently, so the database didn't save the order properly. I'll make a fix to check and auto-correct playlist song order.

**Reply by z-huang** [reply]
In case I forget it, can you create an issue?

**Comment by Snapxxx12** [comment]
Will you add audio fade in & fade out features when songs are played and paused, as well as when songs change? Also, can you apply the fade in & fade out effect when selecting a song?

**Comment by Secretkillar** [comment]
Please add listen together option for long distance gf and bf experience same music. Please 🙏🥺

**Reply by z-huang** [reply]
Sounds interesting! Please create an issue for that.

**Comment by Melo-5** [comment]
Hello Z! 

I'm in love with your app. I was wondering if it's possible to add a simple widget with the same minimalism material you theme as the app. Look at this 2 examples: 

![Screenshot_2024-08-24-10-58-48-820_com microsoft emmx](https://github.com/user-attachments/assets/5711fab2-da33-4a01-a13f-0d7f2f96793b)


Thanks for your time and effort. Let me know how can I buy you a coffee! Keep it up!

**Reply by z-huang** [reply]
Thanks for your support!
What's the app in your screenshot? Is it open-sourced? If so, I can take a look at their implementation.
You can find donation links on the home page https://github.com/z-huang/InnerTune.

**Reply by Melo-5** [reply]
Hello Z and Naoto.

I saw it here: https://www.reddit.com/r/kustom/comments/yqvdh5/my_material_your_kwgt_app_very_alpha_read/?utm_source=share&utm_medium=android_app&utm_name=androidcss&utm_term=1&utm_content=share_button

I think is the same creator.

**Comment by Isssmael2001** [comment]
I can't log in with my Google account. I connect to my Google account, but it stays in that window that opens to log in. When I log out, it doesn't connect.

**Comment by kuchingneko28** [comment]
Does the like button add a song to the favorite songs playlist? Also, I don’t see any history of the songs I've played on my account it seems aren’t being recorded

**Comment by PlasticFoods777** [comment]
🔥🗣️🙏Thanks for providing this cool app!

**Comment by Harsha130-del** [comment]
Having the same problem with Equalizer. It's not opening....do something about it

**Reply by z-huang** [reply]
Cannot reproduce.
Screen recording please.
